test/components/views/rooms/MemberList-test.tsx

```diff
@@ -406,7 +406,7 @@ describe("MemberList", () => {
                 await flushPromises();
 
                 // button rendered but disabled
-                expect(screen.getByText("Invite to this room")).toBeDisabled();
+                expect(screen.getByText("Invite to this room")).toHaveAttribute("aria-disabled", "true");
```

Comment:
I'm a little curious why `toBeDisabled` has stopped working

Reply:
Compound switched from `disabled` to `aria-disabled` and Jest doesn't understand the latter - https://github.com/testing-library/jest-dom/issues/144#issuecomment-577235097